.style-and-markdown-build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
```

Comment:
Is there a particular reason to not set `nounset`? You resolved my suggestion for adding it without comment. I don't think it will have a huge effect here but it also will not hurt and is a safer choice when people potentially modify the script.

```suggestion
set -ue
```

Reply:
Nope, no reason, just forgot to add it :D